### DISCUSSION TITLE
Please i cant edit custom type component

### DISCUSSION THREAD

**Discussion by neutron92** [GrapesJS/grapesjs#5864] [Q&A]
Hello 
I have a 

`<p data-gjs-type=“stylable-item-block”>….`

![IMG_3463](https://github.com/GrapesJS/grapesjs/assets/6135555/2b4e4b3c-1958-41bb-83ba-ebb187c71ace)

but i can’t edit it and its content of text in the browser when i run the project 

any help please?

**Comment by al-sabr** [comment]
Hi there I think there's to much dirt on your monitor if you clean it it will work.😄😃

**Comment by artf** [comment]
I guess you would need to extend the `text` component.
